Source/Plugins/Core/com.equella.core/js/__stories__/components/SearchResult.stories.tsx

```diff
@@ -21,7 +21,7 @@ import SearchResult from "../../tsrc/search/components/SearchResult";
 import { number, object, text } from "@storybook/addon-knobs";
 
 export default {
-  title: "SerachResult",
+  title: "SearchResult",
```

Comment:
```suggestion
  title: "Search/SearchResult",
```
While you're there, let's move this into the new hierarchy. (Ideally, if you want it'd also be good to move from `__stories__/component` to `__stories__/search` too.)